@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -210,13 +204,17 @@ export class TileGeometryLoader {
         if (
             (this.m_state === TileGeometryLoaderState.Initialized ||
                 this.m_state === TileGeometryLoaderState.Canceled) &&
-            tile.decodedTile !== undefined
+            tile.tileLoader?.isFinished === true
         ) {
-            if (this.m_state === TileGeometryLoaderState.Initialized) {
+            if (
+                this.m_state === TileGeometryLoaderState.Initialized &&
+                tile.decodedTile !== undefined
```

Comment:
I wonder if it wouldn't make things easier if we instead of not producing a tile could just create a decoded tile that has no geometry.

Reply:
I think, as long as we wouldnt return a decoded Tile containing the background tile geometry, it does not make much of a difference

Reply:
I will take another look, the decodedTile that comes through is indeed not undefined, and has no geometry, not sure how I missed this, anyway, I think I don't need to change this condition.